vendor/rild.te

```diff
@@ -19,7 +19,9 @@ unix_socket_connect(rild, netmgrd, netmgrd)
 add_hwservice(rild, vnd_ims_radio_hwservice)
 add_hwservice(rild, vnd_qcrilhook_hwservice)
 
+set_prop(rild, vendor_radio_prop)
+
 allow rild self:socket ioctl;
 allowxperm rild self:socket ioctl msm_sock_ipc_ioctls;
 
-allow rild secure_element:binder call;
+hal_server_domain(rild, hal_secure_element)
```

Comment:
Because `allow rild hal_secure_element_hwservice:hwservice_manager add;` is a neverallow I put this into my branch, @alviteri have you copied it from there or can you confirm it actually works?

Reply:
It worked but I will recheck tomorrow